packages/tables/resources/views/columns/color-column.blade.php

```diff
@@ -0,0 +1,44 @@
+@if ($getState())
+    @if ($isCopyable())
+        <button
+            x-data="{
+                showCopied: false,
+                copiedMessageTimeout: -1,
+                copy() {
+                    window
+                        .navigator
+                        .clipboard
+                        .writeText('{{ $getState() }}')
+                        .then(() => {
+                            this.showCopied = true;
+                            window.clearTimeout(this.copiedMessageTimeout);
+                            this.copiedMessageTimeout = window.setTimeout(() => {
+                                this.showCopied = false;
+                            }, {{ $getCopyMessageShowTimeMs() }});
+                        });
+                },
+            }"
+            @click.prevent.stop="copy"
+        >
+            @endif
+            <span
+                class="relative flex"
+                style="background-color: {{ $getState() }}; width: 1.5rem; height: 1.5rem; margin-left: 1rem; border-radius: 6px;"
```

Comment:
Could you move these inline styles (except the background color of course) to Tailwind classes please?

Reply:
This one is resolved.